hypervisor/include/arch/x86/mmu.h

```diff
@@ -233,8 +233,8 @@ uint64_t gpa2hpa(struct acrn_vm *vm, uint64_t gpa);
  * @param[out] size the pointer that returns the page size of
  *                  the page in which the gpa is
  *
- * @return INVALID_HPA - the HPA of parameter gpa is unmapping
- * @return hpa - the HPA of parameter gpa is hpa
+ * @retval INVALID_HPA - the HPA of parameter gpa is unmapping
+ * @return the HPA of parameter gpa is hpa
```

Comment:
Same here (although that function does not appear in the online doc)

Reply:
Updated similarly.